<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewTask</class>
 <widget class="QDialog" name="AddNewTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>417</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>298</width>
    <height>417</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>298</width>
    <height>417</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(220, 235, 255);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBoxName">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-2</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border: 2px solid;
background-color: rgb(255, 235, 187);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>20</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border:1px solid;
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>0</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxTime">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>57</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border: 2px solid;
background-color: rgb(255, 235, 187);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>13</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>От</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEditStartTime">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>12</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateTimeEdit{
border: 1px solid;
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="time">
     <time>
      <hour>10</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>H:mm</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>49</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>По</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEditFinishTime">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>46</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateTimeEdit{
border: 1px solid;
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="time">
     <time>
      <hour>11</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>H:mm</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>80</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Весь день</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxStart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>12</y>
      <width>123</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
border: 1px solid;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDateEdit" name="dateEditStartDate">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="displayFormat">
      <string>ddd dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFinish">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>46</y>
      <width>123</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
border: 1px solid;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDateEdit" name="dateEditFinishDate">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="displayFormat">
      <string>ddd dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxNameRepeat">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>166</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border: 2px solid;
background-color: rgb(255, 235, 187);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Повторить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRepeat">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>26</y>
      <width>105</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxNameRemind">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>225</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border: 2px solid;
background-color: rgb(255, 235, 187);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Напоминание</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRemind">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>26</y>
      <width>105</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxNameDescription">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>284</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border: 2px solid;
background-color: rgb(255, 235, 187);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Описание</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditDescription">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>25</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit{
border:1px solid;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>381</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
background-color: rgb(255, 255, 222);
}</string>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>380</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid;
background-color: rgb(255, 255, 222);
}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
